src/components/App/App.js

```diff
@@ -23,11 +23,39 @@ class App extends Component {
   }
 
   componentDidMount() {
+    window.addEventListener("resize", this.handleResize)
+    this.handleResize();
```

Comment:
Nice job finding a way for the right amount of movies to display when the screen starts small